packages/esm-patient-banner-app/src/banner/patient-banner.component.tsx

```diff
@@ -60,13 +76,26 @@ const PatientBanner: React.FC<PatientBannerProps> = ({ patient, patientUuid, hid
         </div>
         <PatientBannerPatientInfo patient={patient} />
         <div className={styles.buttonCol}>
-          {!hideActionsOverflow ? (
-            <PatientBannerActionsMenu
-              actionsSlotName="patient-actions-slot"
-              isDeceased={patient.deceasedBoolean}
-              patientUuid={patientUuid}
-            />
-          ) : null}
+          <div className={styles.buttonRow}>
```

Comment:
The print button should be moved into the actions menu rather than being a standalone thing here.

Reply:
Do you agree with this suggestion, @jnsereko?

Reply:
Just channeling [Paul's comment here](https://openmrs.slack.com/archives/CKS32D55G/p1716539108400769?thread_ts=1716473583.247999&cid=CKS32D55G).